### ISSUE TITLE
[Bug] Accessibility (Black&White)

### ISSUE THREAD

**Issue by N3TC4T** [XRPL-Labs/XUMM-Issue-Tracker#407]
When using the mobile device's Black & White screen Accessibility feature, I've noted certain segments of the XummWallet that aren't visually obvious to the user due to a lack of contrast.

An example being the input of pincode. Typically (In normal/dark modes) a user can see the dots change colour, however the difference is too subtle in black&white. I haven't tried the other accessibility types but thought this was worth consideration.
